settings.php

```diff
@@ -432,8 +436,8 @@ function get_FTL_data($arg)
                     } else {
                         $DHCP = false;
                         // Try to guess initial settings
-                        if ($piHoleIPv4 !== "unknown") {
-                            $DHCPdomain = explode(".", $piHoleIPv4);
+                        if ($IPv4GW !== "unknown") {
+                            $DHCPdomain = explode(".", $IPv4GW);
                             $DHCPstart = $DHCPdomain[0] . "." . $DHCPdomain[1] . "." . $DHCPdomain[2] . ".201";
                             $DHCPend = $DHCPdomain[0] . "." . $DHCPdomain[1] . "." . $DHCPdomain[2] . ".251";
                             $DHCProuter = $DHCPdomain[0] . "." . $DHCPdomain[1] . "." . $DHCPdomain[2] . ".1";
```

Comment:
Two things here:
- `$DHCPdomain` - "domain" doesn't make much sense here
- `$DHCProuter` - router is nothing else than the gateway in this context so we can just reuse the gateway address instead of assuming the router will be at `.1` (even when this is a really good assumption).
```suggestion
                            $DHCPparts = explode(".", $IPv4GW);
                            $DHCPstart = $DHCPparts[0] . "." . $DHCPparts[1] . "." . $DHCPparts[2] . ".201";
                            $DHCPend = $DHCPparts[0] . "." . $DHCPparts[1] . "." . $DHCPparts[2] . ".251";
                            $DHCProuter = $IPv4GW;
```